### DISCUSSION TITLE
Does not download all images

### DISCUSSION THREAD

**Discussion by rainmakermarketing** [Richard-Weiss/Bing-Creator-Image-Downloader#29] [Q&A]
Bing Image Generator can sometimes generate multiple files for a single prompt. When this happens, and you save the results to Bing IMage Collections, it is stored as a group of (upto 4) images. The problem is, Bing-Creator-Image-Downloader only seems to download one of these images. Usually the first one in the group.

Am I using using it incorrectly or is this a bug?

**Comment by Richard-Weiss** [comment]
@rainmakermarketing
I'm sorry for getting so late to you, it seems like I don't get notifications for new discussions.
Can you explain how you are saving the images to the collection?
I'm not aware of being able to save image sets instead of the 4 individual images.
Can you show me how it looks like in https://www.bing.com/saves for you?
Are really all images added to the collection when you add them?
